feeds/npm/npm.go

```diff
@@ -117,16 +127,34 @@ func (feed Feed) Latest(cutoff time.Time) ([]*feeds.Package, error) {
 		return pkgs, err
 	}
 
+	// Handle the possibility of multiple releases of the same package
+	// within the polled `packages` slice.
+	uniquePackages := make(map[string][]*Package)
 	for _, pkg := range packages {
-		go func(pkg *Package) {
-			v, err := fetchVersionInformation(pkg.Title)
+		v, ok := uniquePackages[pkg.Title]
+		if ok {
```

Comment:
I think this can be tidied with 
```
pkgs, ok := uniquePackages[pkg.Title]
if !ok {
  pkgs = []*Package{}
}
// Insert common append logic here
```

Reply:
Changed the approach with the latest version, but thanks :)